Services/DataBaseService/DbModels/DataBaseContext.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DataBaseService.DbModels;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System.Reflection;
+
+namespace DataBaseService
+{
+    public class DataBaseContext: DbContext
+    {
+        public DbSet<DbUser> Users { get; set; }
```

Comment:
Why not DbUsers? I think it can mislead people

Reply:
> Why not DbUsers? I think it can mislead people

Not sure if `Users` isn't ok because it's in `DataBaseContext` class

Reply:
It is better if it has same name as table in database